File: FrontEnd/web/module/itemDTO.js
function itemDTO(id,name,qty,price){
    var __id =id;
    var __name=name;
    var __qty=qty;
    var __price=price;

    this.getItemId=function (){
        return __id;
    };

    this.setItemId=function (value){
        __id=value;
    };

    this.getItemName=function (){
        return __name;
    };

    this.setItemName=function (value){
        __name=value;
    };

    this.getItemQty=function (){
        return __qty;
    };

    this.setItemQty=function (value){
        __qty=value;
    };

    this.getItemPrice=function (){
        return __price;
    };

    this.setItemPrice=function (value){
        __price=value;
    };
}

File: BackEnd/src/dto/ItemDTO.java
package dto;

public class ItemDTO {
    private String itemId;
    private String itemName;
    private int itemQty;
    private double itemPrice;

    public ItemDTO(String itemId, String itemName, int itemQty, double itemPrice) {
        this.setItemId(itemId);
        this.setItemName(itemName);
        this.setItemQty(itemQty);
        this.setItemPrice(itemPrice);
    }

    public ItemDTO() {
    }


    public String getItemId() {
        return itemId;
    }

    public void setItemId(String itemId) {
        this.itemId = itemId;
    }

    public String getItemName() {
        return itemName;
    }

    public void setItemName(String itemName) {
        this.itemName = itemName;
    }

    public int getItemQty() {
        return itemQty;
    }

    public void setItemQty(int itemQty) {
        this.itemQty = itemQty;
    }

    public double getItemPrice() {
        return itemPrice;
    }

    public void setItemPrice(double itemPrice) {
        this.itemPrice = itemPrice;
    }

    @Override
    public String toString() {
        return "ItemDTO{" +
                "itemId='" + itemId + '\'' +
                ", itemName='" + itemName + '\'' +
                ", itemQty=" + itemQty +
                ", itemPrice=" + itemPrice +
                '}';
    }
}
